src/Hydration.js

```diff
@@ -0,0 +1,50 @@
+class Hydration {
+  constructor(hydrationData) {
+    this.hydrationData = hydrationData;
+  }
+
+  calculateAverageWaterIntake(userID) {
+    const currentUserHydrationData = [];
+    const totalWaterIntake = this.hydrationData.reduce((sum, hydrationInfo) => {
+      if(hydrationInfo.userID === userID) {
+        currentUserHydrationData.push(hydrationInfo)
+        sum += hydrationInfo.numOunces;
+        }
+      return sum;
+    }, 0);
+    return Math.floor(totalWaterIntake / currentUserHydrationData.length);
+  }
```

Comment:
I expect we can make this shorter - maybe consider coming back to refactor, but balance consideration of readability if doing so.